docs/sources/dashboards/previews.md

```diff
@@ -0,0 +1,112 @@
++++
+title = "Dashboard previews"
+keywords = ["grafana", "dashboard", "documentation", "previews"]
+aliases = ["/docs/grafana/latest/reference/previews/"]
+weight = 9
++++
+
+# Previews
+
+{{< figure  max-width="950px" src="/static/img/docs/dashboards/previews.jpg" animated-gif="/static/img/docs/dashboards/previews.gif" >}}
+
+> **Note:** Dashboard previews are available in Grafana 9.0+ as an opt-in beta feature. Please refer to the [permissions]({{< relref "#preview-visibility">}}) section to learn about issues with data source permissions before enabling the feature.
+
+Dashboard previews bring life to all dashboard list pages. Besides looking pretty, they can serve as an overview of all available dashboards, or as a tool to quickly find the right dashboard in case the dashboard names aren't enough on their own.
+
+Grafana users are not forced to use previews. Each user can choose to disable them and go back to the usual look and feel of list views. We save the choice in the browser's local storage to make it persistent across sessions.
```

Comment:
```suggestion
It is an opt-in feature and you can choose to disable previews. Grafana saves the choice in the browser's local storage to make it persistent across sessions.

Reply:
There are two "opt-ins":

1. The feature flag, which is Grafana-instance-wide and accessible only to the admins. Enabling the feature flag is a prerequisite for the crawler process & anyone seeing the previews.
2. UI-toggle with state per grafana user. Each grafana user has the opportunity to disable the feature for themselves, even if the crawler has already collected previews for each dashboard

Not sure how to phrase it, but it's important to put it in the docs as users who dislike the new look of dashboard lists will very much welcome the customizability